magpie/api/schemas.py

```diff
@@ -435,6 +461,9 @@ class TemporaryURL_RequestPathSchema(colander.MappingSchema):
     ServicesTag: "Management of service definitions, children resources and their applicable permissions.",
 }
 
+if get_constant("MAGPIE_NETWORK_MODE", settings_name="magpie.network_mode"):
+    TAG_DESCRIPTIONS[NetworkNodeTag] = "Management of references to other Magpie instances in the network."
```

Comment:
Maybe I don't understand the order here. When does the INI file get parsed? At run time, after the python files are parsed but before the server starts?

Reply:
Until the WSGI app is returned from https://github.com/Ouranosinc/Magpie/blob/1d602eedbe4047fd95c4d2684b5731f4d0600977/magpie/app.py#L98
and actually started by Gunicorn, any intermediate `import ...` operation causes the imported module to be executed immediately before the "current app" that `get_constant` can detect is defined. 

Therefore, any call to `get_constant` at module level, such as in this case, is highly dependent on the order that modules are imported, which could be before or after the app is defined. Something that is working at one point could suddenly start breaking simply by changing imports, and this is hard to track. However, if you use `get_constant` within a function, the "current app" should exist because the function execution is performed during runtime, contrary to its definition.